### PR TITLE
feat: switch to ampli

### PR DESCRIPTION
**Pull request by natapg**
Closes https://github.com/fingerprintjs/fingerprint.com/issues/1341

### CODE REVIEW

File: src/helpers/amplitude.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        }
+
+    ampli.load({
+      client: {
+        apiKey: AMPLITUDE_API_KEY,
+        options: options,
+      },
+    })
+    if (visitorId) {
+      ampli.client?.setDeviceId(visitorId)
```
Comment (dcorso21) [inline src/helpers/amplitude.tsx:33]: Is this already being set as the visitorId on the website? I don't think we set the device id to the visitorId on the dashboard, but I can't remember.

File: src/helpers/amplitude.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React, { createContext, useContext, useEffect } from 'react'
+import { AMPLITUDE_API_KEY, AMPLITUDE_API_ENDPOINT, RANDOM_AMPLI, IS_VERCEL } from '../constants/env'
+import { generateRandomString } from './common'
+import { BASE_URL } from '../constants/content'
+import { ampli, Ampli } from '../ampli'
+import { useVisitorData } from '@fingerprintjs/fingerprintjs-pro-react'
+import { getConfig } from './fpjs'
+
+const AmpliContext = createContext<Ampli | null>(null)
```
Comment (dcorso21) [inline src/helpers/amplitude.tsx:9]: I think you can probably just initialize it with `ampli` right?

File: package.json [hunk 3]
```diff
@@ -42,6 +42,7 @@
     "@rollbar/react": "^0.11.2",
     "@tippyjs/react": "^4.2.0",
     "@vercel/node": "^2.15.2",
+    "ampli": "^0.12.2",
```
Comment (dcorso21) [inline package.json:45]: you may also want to define a script command to pull the latest tracking plan -- specifying the correct source.

**Review by dcorso21 — CHANGES_REQUESTED** [review]

File: src/helpers/amplitude.tsx (hunk 1 above)
Comment (dcorso21) [reply src/helpers/amplitude.tsx:33]: Oh, you're right, I just looked at the dashboard too.

File: src/components/ContactSalesForm/index.tsx [hunk 4]
```diff
@@ -165,13 +166,13 @@ export default function ContactSalesForm({ variant = 'contactSales' }: ContactSa
         switch (variant) {
           case 'getDemo':
             sendFormEvent('GetDemo-module--form')
-            amplitudeLogEvent('Get Demo Contacted')
+            ampli && ampli.client.logEvent('Get Demo Contacted')
```
[the remark this answers is not in the document]
Comment (natapg) [reply src/components/ContactSalesForm/index.tsx]: I tagged you on slack

File: src/helpers/amplitude.tsx (hunk 2 above)
Comment (natapg) [reply src/helpers/amplitude.tsx]: yes :)

File: src/components/ContactSalesForm/index.tsx [hunk 5]
```diff
@@ -165,13 +166,13 @@ export default function ContactSalesForm({ variant = 'contactSales' }: ContactSa
...
             break
           case 'contactSales':
           default:
             trackLeadSubmit()
             sendFormEvent('ContactSalesForm-module--form')
-            amplitudeLogEvent('Sales Contacted')
+            ampli && ampli.salesContacted()
```
[the remark this answers is not in the document]
Comment (natapg) [reply src/components/ContactSalesForm/index.tsx]: my bad,fixed!

File: src/components/Layout/index.tsx [hunk 6]
```diff
@@ -43,14 +43,13 @@ export function LayoutTemplate({ children, darkVariant }: LayoutTemplateProps) {
...
-        visitorId
-      )
+    if (ampli) {
+      ampli.viewMarketingPage({
+        route: pathname,
+        botDetected: visitorData?.products?.botd?.data?.bot?.result ?? 'BotD data not available',
+      })
     }
-  }, [pathname, visitorData, visitorId])
+  }, [pathname, ampli, visitorData])
```
[the remark this answers is not in the document]
Comment (natapg) [reply src/components/Layout/index.tsx]: excellent suggestion

**Review by dcorso21 — APPROVED** [review]

**Pull request merged**